tests/components/unifiprotect/utils.py

```diff
@@ -0,0 +1,172 @@
+"""Test helpers for UniFi Protect."""
+# pylint: disable=protected-access
+from __future__ import annotations
+
+from dataclasses import dataclass
+from datetime import timedelta
+from typing import Any, Callable, Sequence
+
+from pyunifiprotect import ProtectApiClient
+from pyunifiprotect.data import (
+    Bootstrap,
+    Camera,
+    ProtectAdoptableDeviceModel,
+    WSSubscriptionMessage,
+)
+from pyunifiprotect.data.bootstrap import ProtectDeviceRef
+from pyunifiprotect.test_util.anonymize import random_hex
+
+from homeassistant.const import Platform
+from homeassistant.core import HomeAssistant, split_entity_id
+from homeassistant.helpers import entity_registry as er
+from homeassistant.helpers.entity import EntityDescription
+import homeassistant.util.dt as dt_util
+
+from tests.common import MockConfigEntry, async_fire_time_changed
+
+
+@dataclass
+class MockEntityFixture:
```

Comment:
```suggestion
class MockUFPFixture:
```

I thought this was a Home Assistant entity at first glance

Reply:
Not sure that name is better though

Reply:
Well, the old name was misleading because it was basically just a helper class that had the config entry and API in it.

Reply:
Lol. I _completely_ misread what was going on in this comment last night. I guess was a bit tired.